index.html

```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
           cette ville
         </p>
       </section>
+      <!--section : the most popular-->
+      <section>
+        <div id="most_popular">
```

Comment:
remove this div and put id directly on section

Reply:
ok